extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzSchedulerImpl.java

```diff
@@ -581,6 +585,7 @@ private Properties getSchedulerConfigurationProperties(QuartzSupport quartzSuppo
                     quartzSupport.getDriverDialect().get());
             props.put(StdSchedulerFactory.PROP_DATASOURCE_PREFIX + "." + dataSource + ".connectionProvider.class",
                     QuarkusQuartzConnectionPoolProvider.class.getName());
+            props.put(StdSchedulerFactory.PROP_JOB_STORE_PREFIX +".acquireTriggersWithinLock","true");
             if (buildTimeConfig.clustered) {
                 props.put(StdSchedulerFactory.PROP_JOB_STORE_PREFIX + ".isClustered", "true");
                 props.put(StdSchedulerFactory.PROP_JOB_STORE_PREFIX + ".acquireTriggersWithinLock", "true");
```

Comment:
I think we can remove this line (https://github.com/quarkusio/quarkus/blob/88bfa7eccef25b99a47f510ebb99ea679fe48d1a/extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzSchedulerImpl.java#L591), since it is done above https://github.com/quarkusio/quarkus/blob/88bfa7eccef25b99a47f510ebb99ea679fe48d1a/extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzSchedulerImpl.java#L588

Reply:
Done.